web/apps/web/src/components/AppLayout/NavMenu/index.tsx

```diff
@@ -13,8 +13,12 @@ export function NavMenu() {
     const { logout } = useLogoutFlow({ kratosClient, onLoggedOut: () => sessionManager.checkIfLoggedIn() })
 
     const [activeKey] = useKeyByRoute({
-        [MenuKeys.Projects]: useMatch(path("projects", "index")),
+        [MenuKeys.Projects]: [
+            useMatch(path("projects", "index")),
+            useMatch(path("projects", "project", { projectId: "*" })),
+        ],
```

Comment:
```suggestion
        [MenuKeys.Projects]: useMatch(path("projects", "index") + "/*"),
```

? (or just `*` I'm not sure)